app/controllers/AddAdditionalMetadataController.scala

```diff
@@ -101,7 +117,8 @@ class AddClosureMetadataController @Inject() (
       consignmentId: UUID,
       fileIds: List[UUID],
       metadataInput: List[UpdateFileMetadataInput],
-      propertyNameValueSelectedAndDepsToDel: Set[ValueSelectedAndDepsToDel]
+      propertyNameValueSelectedAndDepsToDel: Set[ValueSelectedAndDepsToDel],
```

Comment:
Ah, yes. This is because Scalafmt (that made this change) and Scalastyle disagree with each other. Should I turn off the "if brace" check in scalastyle (if so, just for this particular case or all cases?) or should I find a way to remove this way of formatting if statements from Scalafmt?

Reply:
Leave it for now. Thanks